CancerRelatedProcedures.fsh

```diff
@@ -76,6 +73,19 @@ Description: "The total amount of radiation dose delivered for the course of the
 * valueQuantity = UCUM#cGy
 
 
-
-
-
+Profile:  CancerRelatedSurgicalProcedure
+Parent:   Procedure  // USCoreProcedure
+Id:       CancerRelatedSurgicalProcedure
+Title:    "Cancer-Related Surgical Procedure"
+Description: "A surgical action addressing a cancer condition. The scope of this profile has been narrowed to cancer-related procedures by constraining the ReasonReference and ReasonCode to cancer conditions. Conformance note: If an ICD-10-PCS code is used in the code attribute, and there is a semantically equivalent SNOMED CT or CPT code, the resulting Procedure instance will not be compliant with US Core Profiles."
+* code from CancerRelatedSurgicalProcedureVS (extensible)
+* extension contains 
+    TreatmentIntent 0..1 and 
+    StatementDateTime 0..1
+* reasonCode, reasonReference, extension[TreatmentIntent] MS  // other MS will be inherited from USCoreProcedure
```

Comment:
bodySite and its extensions (Laterality, AnatomicalOrientation, and RelationToLandmark) are missing.  Is this intentional?

Reply:
Good catch, I missed that. I added it in.